e2e/init_start_panacea.sh

```diff
@@ -45,7 +45,7 @@ panacead tx datadeal create-deal /tmp/create_deal.json \
   -b block \
   --yes
 
-panacead tx datapool register-node "https://my-endpoint.com" \
+panacead tx oracle register-node "https://my-endpoint.com" \
   --from oracle \
```

Comment:
We only do key generation with validator. But in our example `oracle, curator, etc.` are used.
How about adding key generation with curator and oracle?